US-CBP/cbp-theme
# Fix/font path issue in non minified css

#### What's this PR do?

In the minified version of the cbp-theme the font paths include a hash like so:
Roboto-Light_1fa8191a.woff2 

resulting in not found issues

when they should look like roboto-fontface/fonts/roboto/Roboto-Light.woff2

#### Where should the reviewer start?
Start with sass-builder.js line 14 and understand the postCss options. useHash specifically

#### How should this be manually tested?

1. Run npm run build and check that the minified and non minified font paths are the same for the .woff(2) files

2. Run npm run dev and test the kitchensink app locally and inspect the fonts. Kitchen sink uses the minified css. Also inspect the dist non minified and minified css and the font paths should be the same. 

#### Any background context you want to provide?

Also, some cleanup using the lint which is automatically run when running **npm run dev**

#### What are the relevant tickets?

See #241 

#### Screenshots (if appropriate)
N/A

#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? N/A
